doc/howtos/image-install.rst

```diff
@@ -86,3 +86,52 @@ includes a checksum verification, you can also use the traditional :command:`dd`
 
 Unplug the removable media from your development system and you're ready to plug
 it into your target system.
+
+Installing to internal Media
```

Comment:
I did that here to be consistent with capitalization in headers elsewhere ("major" words in uppercase, adjectives lowercase - not sure whether that's exactly the rule).

I don't have a strong preference regarding media vs. medium. I used singular when talking about one particular instance and plural when there could be more than one.